src/base/static/components/app.tsx

```diff
@@ -105,6 +105,7 @@ const TemplateContainer = styled("div")<{
   currentTemplate: string;
 }>(props => ({
   position: "relative",
+  "-webkit-overflow-scrolling": "touch", // Inertia-based scrolling on iOS.
```

Comment:
FYI this CSS prefix has nothing to do with iOS - it's for browsers: https://developer.mozilla.org/en-US/docs/Web/CSS/Webkit_Extensions

Also, it looks like this property is deprecated: https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling

Reply:
Ah, I see - that CSS property *only* works on iOS: https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling#Browser_compatibility so I guess it is an iOS-only property....

Reply:
yeah, this caused some other problems. I don't think we should use this property.

Reply:
reverted here: https://github.com/mapseed/platform/commit/33b51b8dde284b941192197c854f288321acb94e

Reply:
Unfortunately `scroll-behavior: touch` doesn't seem to work on Android from what I've tested: https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior